src/GitHubApps/src/DarcBot/DarcBotGitHubClient.cs

```diff
@@ -0,0 +1,145 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Extensions.Logging;
+using Microsoft.IdentityModel.Tokens;
+using Newtonsoft.Json.Linq;
+using Octokit;
+using System;
+using System.IdentityModel.Tokens.Jwt;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Security.Claims;
+using System.Security.Cryptography;
+using System.Threading.Tasks;
+
+namespace DarcBot
+{
+    internal class DarcBotGitHubClient : IGitHubClient
```

Comment:
It's... odd that we are implementing a github client. That shouldn't be necessary. We should have classes that implement the installation stuff already.